hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -853,6 +855,22 @@ public JobTrackerConfig findJobTrackerConfig(String name) {
         return getJobTrackerConfig(name);
     }
 
+    public <T extends CustomServiceConfig> T getServiceConfig(String name) {
```

Comment:
If you are introducing a new type of configuration, I think it should also configurable from XML and Spring.